store/src/java/com/zimbra/cs/service/admin/GetAdminExtensionZimlets.java

```diff
@@ -61,8 +62,23 @@ private void doExtensionZimlets(ZimbraSoapContext zsc, Map<String, Object> conte
 			
 			if (z.isExtension()) {
 			    boolean include = true;
+			    boolean isNGEnabled = true;
+			    try {
+			        isNGEnabled = Provisioning.getInstance().getLocalServer().isNetworkModulesNGEnabled();
+                } catch (ServiceException e) {
```

Comment:
nit: it appears there are some indentation/whitespace issues here and below - maybe tabs vs spaces...